src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -1,54 +1,138 @@
 package org.broadinstitute.consent.http.models;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.CaseFormat;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
 
 public class DataAccessRequest {
 
-    @JsonProperty
-    public Integer id;
+  private static final List<String> DEPRECATED_PROPS = Arrays.asList("referenceId", "investigator",
+      "institution", "department", "address1", "city", "zipcode", "zipCode", "state", "country",
+      "researcher", "userId", "isThePi", "havePi", "piEmail", "profileName", "pubmedId",
+      "scientificUrl", "urlDAA", "nameDAA", "eraExpiration", "academicEmail", "eraAuthorized",
+      "nihUsername", "linkedIn", "orcid", "researcherGate", "datasetDetail", "datasets",
+      "datasetId", "validRestriction", "translatedUseRestriction", "createDate", "sortDate");
 
-    @JsonProperty
-    public String referenceId;
+  private static final Gson GSON = new Gson();
 
-    @JsonProperty
-    public DataAccessRequestData data;
+  @JsonProperty public Integer id;
 
-    @JsonProperty
-    public Boolean draft;
+  @JsonProperty public String referenceId;
 
-    public DataAccessRequest() {
-    }
+  @JsonProperty public DataAccessRequestData data;
 
-    public Integer getId() {
-        return id;
-    }
+  @JsonProperty public Boolean draft;
 
-    public void setId(Integer id) {
-        this.id = id;
-    }
+  @JsonProperty public Integer userId;
```

Comment:
```userId``` is listed within ```DEPRECATED_PROPS```. Should it be assigned as a JsonProperty?

Reply:
The deprecated props are deprecated from `DataAccessRequestData`. I think I should move the deprecated json props to that object instead to avoid confusion.